client/dashboard/sites/settings/site-actions.tsx

```diff
@@ -0,0 +1,88 @@
+import { useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHeading as Heading,
+	Button,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { restoreSitePlanSoftwareMutation } from '../../app/queries';
+import { ActionList } from '../../components/action-list';
+import { DotcomFeatures } from '../../data/constants';
+import type { Site } from '../../data/types';
+
+const useRestorePlanSoftware = ( { slug, options }: Site ) => {
+	const mutation = useMutation( restoreSitePlanSoftwareMutation( slug ) );
+
+	if ( ! options.is_wpcom_atomic ) {
```

Comment:
It would be better to check "plan.features" rather than specific site flags. We might have a lot of different "type of sites" and we'd want to enable/disable features elsewhere rather than doing it at the component level.

Reply:
Sounds good! Let’s try it out and see if removing the check causes any regressions.

Reply:
Hmm... but on WP.com, a site might have the feature without being transferred to Atomic. In that case, we still wouldn't be able to restore themes and plugins...

Reply:
I'm ok with keeping this flag for now. I'm just thinking that ideally in components, we should be testing "type of site", we should be eating "features availability" and in order to do that properly, we need the REST API to be done properly.

Reply:
I think this is fine for now. Semantically, restoring themes & plugins is indeed a feature that is inherent to a site being a WoA site.

Reply:
> Semantically, restoring themes & plugins is indeed a feature that is inherent to a site being a WoA site.

Sure, but what if tomorrow we have a new kind of WoA sites, maybe using a different wp-cloud client (like a commerce dedicated client, what if we support pressable sites...). These things can change but the features won't.

Reply:
Yeah, but just because a site has the feature doesn’t necessarily mean it’s available. Some features still need to be explicitly activated before they can be used. In this case, `is_wpcom_atomic` is really just a proxy to check whether the feature for installing themes and plugins is activated. Maybe in the future, it would make more sense to have something like `isFeatureActivated` for clarity.